ui/minimal/index.js

```diff
@@ -0,0 +1,34 @@
+const app = require('express')();
```

Comment:
There is convention in node to put server code into a file called `server.js`. This can be then launched with `npm start` command.